plugins/plugin-installer/plugin_installer/tests/mock-with-install-args-plugin/requirements.txt

```diff
@@ -0,0 +1 @@
+https://github.com/cloudify-cosmo/softlayer-python/archive/master.zip
```

Comment:
i think we plan on deleting this repo soon no? since we dont need this fork anymore...
maybe use cloudify-rest-client? its pretty light and should install quickly